### DISCUSSION TITLE
Custom channel beta priors with different distributions

### DISCUSSION THREAD

**Discussion by raz1470** [pymc-labs/pymc-marketing#1045] [Q&A] [answered]
I am trying to work out if you can supply custom channel beta priors with different distributions using the MMM class....

For example, if I had 3 channels, and on 2 of them I wanted to supply a HalfNorm prior and on 1 of them I wanted to supply a Normal prior - Can I do this by changing the default_model_config?

I have gone through the code but couldn't work it out.

**Comment by williambdean** [comment, marked as answer]
Hi,

You can use a truncated normal distribution and have the lower bound as -np.inf for "Normal" and lower of 0 for "HalfNormal"

For additional support, that currently isnt built out
